### DISCUSSION TITLE
Documentation resource

### DISCUSSION THREAD

**Discussion by PetroZaburko** [noxoua/filament-activity-log#78] [Q&A] [answered]
Hi! Are there any another resource with documentation, because your site is curently unavailable?

**Comment by noxoua** [comment, marked as answer]
Hello. Thank you for reporting an error on the site. 🙏
We will fix this in the near future. 
In the meantime, you can check the readme files in the [./docs](https://github.com/noxoua/filament-activity-log/blob/main/docs/index.md) folder

**Reply by PetroZaburko** [reply]
Thanks for the quick reply. Everything is fine now.
